### PR TITLE
Improve unittest for std.range.cycle for better documentation.

### PR DESCRIPTION
**Pull request by drug007**
Add all imports needed to run this unittest as a separate code.
Add detailed description of unittest.
Improved readablitity (removed slicing of static arrays and rearranged functions calls).

### CODE REVIEW

File: std/range/package.d [hunk 1]
```diff
@@ -2836,8 +2836,14 @@ Cycle!R cycle(R)(R input)
...
     import std.algorithm : equal;
-
-    assert(equal(take(cycle([1, 2][]), 5), [ 1, 2, 1, 2, 1 ][]));
+    import std.range : cycle, take;
+
+    // Here we create an infinitive cyclic sequence from [1, 2]
+    // (i.e. get here [1, 2, 1, 2, 1, 2 and so on]) then
+    // take 5 elements of this sequence (so we have [1, 2, 1, 2, 1]) 
+    // and compare it with specific values - they should
+    // be equal of course.
```
Comment (burner) [inline std/range/package.d]: "they should be equal of course" sound to informal IMO.

what about, "... and compare them with the expected values for equality" ?

Comment (drug007) [inline std/range/package.d]: I agree, your phrase is more suitable.

**Comment by burner** [comment]
LGTM

**Comment by MartinNowak** [comment]
Auto-merge toggled on

**Pull request merged**